app/PropositionFactory.php

```diff
@@ -67,6 +71,9 @@ public function getFlagCount($id) {
 	public function getFlagTypeCount($id, $type) {
 		return Flags::where('proposition', [$id])->where('type', $type)->count();
 	}
+	public function getGlobalFlagCount() {
+		return Flags::count();
```

Comment:
@pavril So what should this be? `Flags::where('status', '<', 3)->whereNotIn('proposer_id', [$id])->groupBy('proposition')->join('propositions', 'flags.proposition', '=', 'propositions.propositionId')->count()`? Something like that?